### PR TITLE
[TT-1845]

### PR DESCRIPTION
**Pull request by AMM55**
Body Transform documentation should be improved on "automated" JSON <-> XML transformation

### CODE REVIEW

File: tyk-docs/content/advanced-configuration/transform-traffic/request-body.md [hunk 1]
```diff
@@ -163,6 +163,14 @@ So your final JSON output will look much better:
 {"foobar": {"type": "int", "value": "1"}`
 ```
 
+Tyk also provides an `xmlMarshal` template to perform conversions in the opposit direction from JSON to XML.
```
Comment (rewsmith) [inline tyk-docs/content/advanced-configuration/transform-traffic/request-body.md]: fix typo `opposit` and add comma after direction
i.e. Tyk also provides an `xmlMarshal` template to perform conversions in the opposite direction, from JSON to XML.

**Review by rewsmith — CHANGES_REQUESTED** [review]

**Review by rewsmith — APPROVED** [review]

**Review by marksou — APPROVED** [review]

**Pull request merged**